<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerViewTr</class>
 <widget class="QWidget" name="LayerViewTr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>檔案</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原檔大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>處理後大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>狀態</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>詳細</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原檔路徑</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>處理後路徑</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>519</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbAll">
           <property name="text">
            <string>全部</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbUndone">
           <property name="text">
            <string>等待處理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbCheck">
           <property name="text">
            <string>已處理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbUncheck">
           <property name="text">
            <string>未處理</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
